src/api/spec/controllers/webui/cloud/azure/configurations_controller_spec.rb

```diff
@@ -8,24 +8,66 @@
 
   describe 'GET #show' do
     context 'without Azure configuration' do
+      before do
+        Feature.run_with_activated(:cloud_upload, :cloud_upload_azure) do
+          get :show
+        end
+      end
+
       it 'creates an Azure configuration' do
+        expect(assigns(:azure_configuration)).not_to be_nil
       end
     end
 
     context 'with Azure configuration' do
+      let!(:azure_configuration) { create(:azure_configuration, user: user) }
+
+      before do
+        Feature.run_with_activated(:cloud_upload, :cloud_upload_azure) do
+          get :show
+        end
+      end
+
+      it { expect(assigns(:azure_configuration)).to eq(azure_configuration) }
     end
   end
 
   describe 'PUT #update' do
+    let!(:azure_configuration) { create(:azure_configuration, user: user) }
+
     context 'with valid parameters' do
+      before do
+        Feature.run_with_activated(:cloud_upload, :cloud_upload_azure) do
+          put :update, params: { cloud_azure_configuration: { application_id: 'random_string' } }
+        end
+        azure_configuration.reload
+      end
+
+      it { expect(flash[:success]).not_to be_nil }
     end
 
     context 'with invalid parameters' do
+      before do
+        Feature.run_with_activated(:cloud_upload, :cloud_upload_azure) do
+          put :update, params: { cloud_azure_configuration: { application_id: '' } }
+        end
+        azure_configuration.reload
+      end
+
+      it { expect(flash[:error]).not_to be_nil }
```

Comment:
The ` 'random_string' ` in the previous test should also be invalid. Are you sure that request succeeds?

Reply:
@bgeuken 

> The 'random_string' in the previous test should also be invalid (assuming it's active record). 

From the comments in the Cloud::Azure::Configuration model:

```
#  application_id  :text(65535)
#  application_key :text(65535)
```

> Are you sure that request succeeds?

Yes. I checked before submitting the commit. Anyway let's wait for travis.

Reply:
Interesting, but ok